test-lib.sh

```diff
@@ -221,10 +235,11 @@ function ct_npm_works() {
     return 1
   fi
 
-  docker run --rm ${IMAGE_NAME} /bin/bash -c "npm install jquery && test -f node_modules/jquery/src/jquery.js"
+  docker run $(ct_mount_ca_file) --rm ${IMAGE_NAME} /bin/bash -c "npm --verbose install jquery && test -f node_modules/jquery/src/jquery.js"
+
   if [ $? -ne 0 ] ; then
-    echo "ERROR: npm could not install jquery inside the image ${IMAGE_NAME}." >&2
-    return 1
+        echo "ERROR: npm could not install jquery inside the image ${IMAGE_NAME}." >&2
+        return 1
```

Comment:
Random indent change?

Reply:
Caused by PyCharm. Will fix it in next PR.